### ISSUE TITLE
Request reserved namespace prefix "ibm"

### ISSUE THREAD

**Issue by mrutkows** [CycloneDX/cyclonedx-property-taxonomy#34]
On behalf of IBM, I kindly request to reserve the namespace 'ibm' for use in IBM produced public SBOMs.

## Namespace to be reserved

| Namespace | Description | URN equivalent |
| :---  | :---  | :---  |
| `ibm` | Namespace for use in IBM SBOM's  within property `name` field values | `urn:ibm:names` |

### Planned usage

Currently, we plan to use this as an alias (short form) for our current URN (i.e., `urn:ibm:names`) to add/associate domain specific information such as:
- IBM product and deliverable IDs to `component` properties
- additional scan/internal tooling/classification information to `component` properties
- risk and confidence scoring information to `component` properties
- legal (usage, coverage) disclaimers to `metadata` properties for customers

### Goal: Simplification

Reduction in length of property name value:

```json 
{
"name": "urn:ibm:names:identifier:product",
"value": "5737-I23"
} 
```

would be simplified to:
```json
{
"name": "ibm:identifier:product",
"value": "5737-I23"
} 
```
**Note** It is my hope we support URN syntax as well (and update the ABNF for package names correspondingly). See issue: https://github.com/CycloneDX/cyclonedx-property-taxonomy/issues/9

**Comment by stevespringett** [comment]
@coderpatros How do you want to handle urns?

IMO, the urn should be optional as to provide backward compatibility, but should be recommended. Thoughts?

**Comment by coderpatros** [comment]
I think we should reserve the usage of the "urn" namespace for URNs. I can update the ABNF to support both.

**Comment by jkowalleck** [comment]
@coderpatros no need to update the ABNF, as long as the namespace is reserved, i guess. will open a PR soon.

regarding urn support in ABNF - this would be an own topic, right? see #9

**Comment by jkowalleck** [comment]
@mrutkows ,

the Top-Level namespace `ibm` was reserved a while ago. 
Could you add the respective taxonomy documentation?

**Comment by mrutkows** [comment]
@jkowalleck I really have no idea what a "published doc-space" mention in #109 is nor what taxonomy docs. are needed. Can you provide me references for what you are asking for?

**Comment by jkowalleck** [comment]
@mrutkows , 

the namespace `ibm` was registered and `RESERVED`.

Reserving a namespace is okay for a while, but what purpose has a registered namespace, if it is unclear how it is used? To solve this, it is intended to publish the taxonomy behind that reserved namespace - usually in a doc-space - like the one for `siemens` <https://github.com/siemens/cyclonedx-property-taxonomy#readme>, or `amazon`'s <https://docs.aws.amazon.com/inspector/latest/user/cyclonedx-namespace.html>.
Even an empty doc-space would be sufficient -- like `dependency-track`'s <https://github.com/DependencyTrack/cyclonedx-property-taxonomy>.
